packages/smarthr-ui/src/components/SideNav/SideNavItemButton.tsx

```diff
@@ -51,30 +51,28 @@ const sideNavItem = tv({
   },
 })
 
-export const SideNavItemButton: FC<Props> = ({
-  id,
-  title,
-  prefix,
-  isSelected = false,
-  size,
-  onClick,
-}) => {
-  const handleClick = onClick
-    ? (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => onClick(e, id)
-    : undefined
+export const SideNavItemButton: FC<Props> = ({ id, title, prefix, isSelected, size, onClick }) => {
+  const handleClick = useMemo(
+    () =>
+      onClick
+        ? (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => onClick(e, e.currentTarget.value)
+        : undefined,
+    [onClick],
+  )
 
   const { wrapperStyle, buttonStyle, buttonInnerStyle } = useMemo(() => {
     const { wrapper, button, buttonInner } = sideNavItem()
+
     return {
-      wrapperStyle: wrapper({ selected: isSelected }),
+      wrapperStyle: wrapper({ selected: isSelected || false }),
```

Comment:
以下でも同じ結果が求められますが、意図が伝わりづらいので ^ の書き方が良さそうでした 👍 
```suggestion
      wrapperStyle: wrapper({ selected: !!isSelected }),
```

Reply:
あー、いや、俺bisonさんが書いてる方法のほうが好きだな...
取り込んじゃいます